.github/workflows/third_party_licenses_check.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: newrelic/rust-licenses-noticer@refactor/code-cleanup
+      - uses: newrelic/rust-licenses-noticer@main
         with:
           template-file: "./THIRD_PARTY_NOTICES.md.tmpl"
-          rust-licenses-noticer-branch: "refactor/code-cleanup"
+          rust-licenses-noticer-branch: ${{ github.ref_name }}
```

Comment:
You are checking the current branch with the code in main. Would it make sense to also check the current branch against itself?

```
- name: Use local hello-world-action
   uses: ./action
   with:
      template-file: "./THIRD_PARTY_NOTICES.md.tmpl"
      rust-licenses-noticer-branch: ${{ github.ref_name }}
```

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

Reply:
Ooh wasn't aware you could do that. Thanks for the heads up! I'm adding this to this PR.

Reply:
Done in https://github.com/newrelic/rust-licenses-noticer/pull/4/commits/8a470914e819822eff4cc35404ba682e72d775ef!

Reply:
Is there value in having both checks?

- current branch against itself
- current branch against main

Maybe it just makes things harder if we want a breaking change against main.

Reply:
Hmm, not sure. This workflow is not exactly a testing pipeline for this project. It is to maintain the third party licenses file updated, but it also happens to perform this check "with itself". I think there's value on checking each branch with the state of that same branch, but it could also work by using the `main` versions of both (action and code).

I'm fine with both approaches, but I wouldn't use more than one. What do you think?

Reply:
Okey, I would proceed then. I feel like checking against itself is better.